packages/hooks/src/useDisclaimer.js

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react';
```

Comment:
I wonder if we need this hook. The disclaimers api was intended to be deprecated in favor of spaces. Exposing this hook might be confusing. Instead, I think we might want to encourage people to use `@availity/spaces`. Keep in mind, `sdk-platform-disclaimers` reads from same data store as `spaces` now. Thoughts? @robmcguinness @GoPro16 

https://github.com/Availity/availity-react/blob/master/packages/spaces/src/SpacesDisclaimer.js#L9

Reply:
So this would work, however there are cases were we don't have the disclaimer but we have a `disclaimerId` as a piece of metadata on the field itself. So you would have two instances of `useSpace` being called.

```javascript
const { space = {} } = useSpace(spaceId);
const { space: disclaimer = {} } = useSpace(get(space,'metdata.disclaimerId`)):
```

This approach is fine but we just are going to have to make a change to the `@availity/spaces` package to fetch the `space` if it doesn't exist.

Then we may have to make a decision on if this is the default behavior or if we need to start adding an `options` param to the `useSpace` or `SpacesProvider` for `fetchingIfNotExist`. We would also have to keep track of the ids that we passed in the `SpacesProvider` props so that we don't refetch on ones we have already tried.

Reply:
I don't think the tradeoff in complexity is worth it. Why not just require that the consumer of the spaces package pass in `space.metadata.disclaimerId` in the `spaceIds` prop of the `SpacesProvider` ?

We already fetch any new spaces that get passed in: https://github.com/Availity/availity-react/blob/master/packages/spaces/src/Spaces.js#L125, so if the consumer needs the initial space first to get the `space.metadata.disclaimerId`, we support that use case.

Reply:
Not sure if I am following completely here. Are you saying to just have another `SpacesProvider` tag nested that will be the one that renders the disclaimer? Putting the newly fetched `disclaimerId` back in the root `SpacesProvider` would assume there is some prop passed down for updating the list of `spaceIds` and payerIds` that was originally passed into that root provider.

Reply:
Deleted `useDisclaimer` and added `SpacesAgreement` component.